README.md

```diff
@@ -1 +1,264 @@
-libsyscall_intercept - coming soon!
+# syscall_intercept
+
+Userspace syscall intercepting library.
+This repository is soon going to be deleted
+
+# How to build #
+
+Building libsyscall_intercept requires cmake.
+Example:
+```sh
+cmake path_to_syscall_intercept -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=clang
```

Comment:
Isn't there a default cmake? So, the minimum might be: 'cmake .'? Also, the convention in other cmake projects is to have a 'build' directory, with specific build types underneath. For example: build/release. I think building libraries in the top level directory isn't good practice. I would like it to follow more of the norm.